lib/3scale/backend/service.rb

```diff
@@ -237,6 +232,18 @@ def active?
         state == :active
       end
 
+      def state=(value)
+        # when nil, then :active
+        value_sym = if value.nil? then :active
+                    else value.to_sym
+                    end
+        # anything but :active will be suspended
+        @state = case value_sym
+                 when :active then :active
+                 else :suspended
+                 end
+      end
```

Comment:
this is pretty verbose and complex, this below is shorter and clear:
```ruby
@state = if value.nil? || value.to_sym != :active
           :suspended
         else
           :active
         end
```
or a more terse version `@state = value.nil? || value.to_sym != :active ? :suspended : :active` or even the slightly different but just as fine `@state = value && value.to_sym == :active ? :active : :suspended`